build-system/tasks/compile-expr.js

```diff
@@ -16,18 +16,61 @@
 'use strict';
 
 const fs = require('fs-extra');
+const glob = require('glob');
 const jison = require('jison');
+const path = require('path');
+const {endBuildStep} = require('./helpers');
+const {jisonPaths} = require('../config');
 
+// set imports for each parser from directory build/parsers/
+const imports = new Map();
+imports.set(
+  'cssParser',
+  "import * as ast from '../../extensions/amp-animation/0.1/parsers/css-expr-ast';"
+);
+imports.set(
+  'bindParser',
+  "import {AstNode, AstNodeType} from '../../extensions/amp-bind/0.1/bind-expr-defines';\n" +
+    "import {tryParseJson} from '../../src/json';"
+);
+
+/**
+ * Builds parsers for extensions with *.jison files.
+ * Uses jison file path to name the parser to export.
+ * For example, css-expr-impl.jison creates `cssParser`
+ * @return {!Promise<void>}
+ */
+function compileExprs() {
+  const startTime = Date.now();
+  fs.mkdirSync('build/parsers', {recursive: true});
+  const promises = [];
+  jisonPaths.forEach(jisonPath => {
+    glob.sync(jisonPath).forEach(jisonFile => {
+      const jsFile = path.basename(jisonFile, '.jison');
+      const extension = jsFile.replace('-expr-impl', '');
+      const parser = extension + 'Parser';
```

Comment:
To inform users of the magic here, should we output a log statement that `foo-expr-impl.jison` was compiled to `build/parsers/fooParser.js`?

Reply:
Yup good idea, new log looks like this
```
Compiled access-expr-impl.jison → build/parsers/access-expr-impl.js as accessParser (438 ms)
Compiled bind-expr-impl.jison → build/parsers/bind-expr-impl.js as bindParser (439 ms)
Compiled css-expr-impl.jison → build/parsers/css-expr-impl.js as cssParser (439 ms)
```